### ISSUE TITLE
Cannot Close Loki-wallet-gui.exe

### ISSUE THREAD

**Issue by flash25296** [loki-project/loki-gui#97]

**Comment by flash25296** [comment]
using GUI version a780896 (Qt 5.12.2) Embedded Loki Version v3.0.2-7-g75c521dd on Win10 Pro I can not close the wallet without opening task manager and killing the task under Processes.

**Comment by Doy-lee** [comment]
Are you using a remote daemon or local? If local, then this could be the daemon is upgrading the DB in the background.

**Comment by dcutugno** [comment]
i'm using a remote node, but have same issue, i have to kill the process

**Comment by flash25296** [comment]
Yes I am also using a remote node as well. I was advised to use the new Beta that is on Github. I have not had a chance to download and test it yet though.
